index.js

```diff
@@ -0,0 +1,5 @@
+function getRandomNumber() {
+  console.log(Math.floor(Math.random() * 1000));
+}
+
+export default getRandomNumber();
```

Comment:
It is ok but, normally you export a function without executing it, like: `export default getRandomNumber;`, you could add the execution of this function after it is exported.